html/web3/cgi-bin/info/hello.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python2
```

Comment:
In my machine, python2 is also shorthand for python2.7. Is it always the case?

Reply:
Usually, there is a symbolic link from python2 to whatever version of python2 is installed in the system.

`ls -l $(which python2)`